HelpSource/Classes/UnitTest.schelp

```diff
@@ -82,98 +82,133 @@ TestUnitTest.new.run;
-::
+argument:: message
+A message describing the purpose of the assertion, e.g. code::"foo
+should be less than bar"::.
+Posted if code::report:: is true.
 
+argument:: report
+Reports the result of the test if code::true:: (default is code::true::)
 
-Make sure to implement the methods code::setUp:: and code::tearDown::. They will be called before and after the test.
+argument:: onFailure
+If not code::nil::, a failure stops the tests and evaluates this function.
 
-METHOD:: assert
+METHOD:: assertEquals
 
-argument:: test
-Make sure that this returns a boolean, where code::true:: means that the test is passed.
+Asserts that an expression code::a:: is equal to the value code::b::,
+where equality is determined according to code::a::'s implementation
+of code::==::.
+
+code::
+this.assertEquals(2 + 2, 4, "passes");
+this.assertEquals(2 + 2, 5, "fails");
+::
+
+argument:: a
+the experimentally produced value
+
+argument:: b
+the expected value
 
 argument:: message
-posted if report is true.
+A message describing the purpose of the assertion, e.g.
+code::"Adding two numbers should return their sum."::.
+Posted if code::report:: is true.
+
+argument:: report
+Reports the result of the test if code::true:: (default is code::true::)
 
 argument:: onFailure
 If not code::nil::, a failure stops the tests and evaluates this function.
 
-code::
-UnitTest.new.assert(2 == 2, "Two does equal two.");
-UnitTest.new.assert(2 == 2.00001, "Two does equal two.");
-::
-
 METHOD:: assertFloatEquals
 
-Make sure that the two floats code::a:: and code::b:: equal within a given range (code::within::).
+Asserts that an expression code::a:: returning a float is within a
+given range (code::within::) of being equal to code::b::.
 
+code::
+this.assertFloatEquals(2, 2.0001, "Pass since 2 is close enough to 2.0001.", 0.001);
+this.assertFloatEquals(2, 2.0001, "Fail since 2 isn't close enough to 2.0001.", 0.0001);
+::
 
 argument:: a
-a float
+the experimentally produced float value
 
 argument:: b
-a float
+the expected float value
 
 argument:: message
-posted if report is true.
+A message describing the purpose of the assertion, e.g.
+code::"Adding two numbers should return their sum."::.
+Posted if code::report:: is true.
+
+argument:: within
+The range within which code::a:: must be of code::b:: in
+order for the test to pass.
+
+argument:: report
+Reports the result of the test if code::true:: (default is code::true::)
 
 argument:: onFailure
 If not code::nil::, a failure stops the tests and evaluates this function.
 
 
+METHOD:: assertArrayFloatEquals
+Make sure that the two arrays of floats code::a:: and code::b:: equal within a given range (code::within::).
 
 code::
-UnitTest.new.assertFloatEquals(2, 2.00001, "Two does equal two.", 0.00001);
-UnitTest.new.assertFloatEquals(2, 2.001, "Two does equal two.", 0.0001);
+this.assertArrayFloatEquals([2, 3] + 0.0001, [2, 3], "Pass since pairs are close enough", 0.001);
```

Comment:
based on the line below I think this should probably be "Pass since floats are close enough"